src/Core/ModulesApplication.php

```diff
@@ -189,7 +189,14 @@ public static function filterSafeLocalModuleFiles(array $files): array
                 // scripts that have any kind of parameters in them such as a cache buster mess up finding the real path
                 // we need to strip that out and then check against the real path
                 $scriptSrcPath = parse_url($scriptSrc, PHP_URL_PATH);
-                $realPath = realpath($GLOBALS['fileroot'] . $scriptSrcPath);
+                // need to remove the web root as that is included in the $scriptSrc and also in the fileroot
+                $pos = strpos($scriptSrcPath, $GLOBALS['web_root']);
```

Comment:
You think for safety should use stripos? Windows is case sensitive.

Reply:
@sjpadgett You mean case insensitive right?  Could this introduce subtle errors where the script doesn't really exist on linux due to to a misnamed filename?  Yet, in the same vein could create problems on windows if the web server path is misconfigured when the windows file system doesn't care.  

I'm fine with using stripos, seems like either way we can have different problems.

Reply:
You're right Windows doesn't care but window folks get use to using caps in paths and filenames.
Do what you think best.